apps/console/src/features/users/utils/user-management-utils.ts

```diff
@@ -24,12 +24,11 @@ import { getUserNameWithoutDomain } from "@wso2is/core/helpers";
 export class UserManagementUtils {
 
     /**
-     * Private constructor to avoid object instantiation from outside
-     * the class.
+     * Remove Private constructor to extend the class
      *
      */
     // eslint-disable-next-line @typescript-eslint/no-empty-function
-    private constructor() { }
+    constructor() { }
```

Comment:
Lets remove the constructor all to gether.

Reply:
Fixed with 49d5472fd021951fadff2b5d758f209c3d22c50c